kbase-extension/static/kbase/css/kbaseJobLog.css

```diff
@@ -40,11 +40,6 @@
   white-space: nowrap;
 
   /* makes text unselectable */
-  -moz-user-select: -moz-none;
-  -ms-user-select: none;
-  -khtml-user-select: none;
-  -webkit-user-select: none;
-  -o-user-select: none;
```

Comment:
is `user-select` the equivalent of the removed styles?

Reply:
Yep.